{"name": "Попытка входа с невалидными данными: неверный пароль", "status": "passed", "steps": [{"name": "Подготовка тестовых данных для сценария: неверный пароль", "status": "passed", "start": 1758449609844, "stop": 1758449609844}, {"name": "Отправляем POST-запрос для авторизации курьера", "status": "passed", "start": 1758449609844, "stop": 1758449610373}, {"name": "Проверяем код ответа 404", "status": "passed", "start": 1758449610373, "stop": 1758449610373}, {"name": "Проверяем сообщение об ошибке", "status": "passed", "start": 1758449610373, "stop": 1758449610373}], "parameters": [{"name": "test_case", "value": "'неверный пароль'"}, {"name": "credentials", "value": "<function TestCourierLogin.<lambda> at 0x000001F23EA09300>"}, {"name": "expected_status", "value": "404"}, {"name": "expected_message", "value": "'Учетная запись не найдена'"}], "start": 1758449609844, "stop": 1758449610374, "uuid": "68d1fc35-70a1-43c7-93ca-9f9f72f991f4", "historyId": "8dba181c2be11c484a4102799e3b3668", "testCaseId": "505bba1c9113ea857d47e178537c8295", "fullName": "tests.test_courier_login.TestCourierLogin#test_login_negative_cases", "labels": [{"name": "story", "value": "Неуспешная авторизация"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "What-Is-Lowe"}, {"name": "thread", "value": "10180-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}], "titlePath": ["tests", "test_courier_login.py", "TestCourierLogin"]}